src/KawaBot.Tests/BasicControllerTests.fs

```diff
@@ -8,15 +8,20 @@ open NUnit.Framework
 [<TestFixture>]
 [<Category(Categories.Integration)>]
 type BasicControllerTests =
+    val private _port: int
```

Comment:
We don't need to save this info. If we'll need it, it's better to add a Port property to the server.